js/data.js

```diff
@@ -1,12 +1,62 @@
-import {getRandomIntNumber} from './util.js';
-function createData() {
-
-  return {
-    id: getRandomIntNumber(1,25),
-    url: `photos/${getRandomIntNumber(1,25)}.jpg`,
-    description: 'Описание фотографии',
-    likes: getRandomIntNumber(15,200),
-    comments: getRandomIntNumber(0,200)
-  };
+const ALERT_SHOW_TIME = 5000;
+
+function getDataFromServer( onSuccess ) {
+  fetch('https://27.javascript.pages.academy/kekstagram-simple/data')
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error(response.statusText);
+      }
+      return response.json();
+    })
+    .then((data) => {
```

Comment:
Если аргумент из колбека такой же как и в обработчике, можно обработчик сразу передать.
```suggestion
    .then(onSuccess)
```